packages/devtools_app/lib/src/screens/memory/panes/diff/controller/utils.dart

```diff
@@ -38,3 +39,22 @@ String classesToCsv(HeapClasses classes) {
 
   return csvBuffer.toString();
 }
+
+String? _cashedIsolateId;
+String? _cashedRootPackage;
+
+/// Returns root library or empty string.
+Future<String?> tryToDetectRootPackage() async {
```

Comment:
is this code that we can share with the debugger page?

Reply:
reused in inspector. Debugger already has isolate instance that it uses for other goals, so it is better as it is.